src/QMCHamiltonians/ForwardWalking.cpp

```diff
@@ -24,18 +24,66 @@
 
 namespace qmcplusplus
 {
+ForwardWalking::ForwardWalking() { update_mode_.set(OPTIMIZABLE, 1); }
+
+ForwardWalking::~ForwardWalking() {}
+
+std::string ForwardWalking::getClassName() const { return "ForwardWalking"; }
```

Comment:
Please keep this defined in the header file.

Reply:
done